soul-admin/src/main/java/org/dromara/soul/admin/controller/DashboardUserController.java

```diff
@@ -110,17 +110,22 @@ public SoulAdminResult detailDashboardUser(@PathVariable("id") final String id)
      */
     @PostMapping("")
     public SoulAdminResult createDashboardUser(@RequestBody final DashboardUserDTO dashboardUserDTO) {
-        String key = secretProperties.getKey();
+        final String key = secretProperties.getKey();
         return Optional.ofNullable(dashboardUserDTO).map(item -> {
             item.setPassword(AesUtils.aesEncryption(item.getPassword(), key));
-            Integer createCount = dashboardUserService.createOrUpdate(item);
+            int createCount = dashboardUserService.createOrUpdate(item);
```

Comment:
why

Reply:
the return data type of member method  "createOrUpdate"  is primitive data type "int".so I think we shouldn't convert to wrapper data type "Integer".